src/javascript/app/common/__tests__/get_app_details.js

```diff
@@ -22,7 +23,7 @@ describe('GetAppDetails', () => {
             expect(GetAppDetails.addAppIdName(2, 'Binary.com')).to.equal('Transaction performed by Binary.com (App ID: 2)');
         });
         it('doesn\'t show for same app id as current', () => {
-            expect(GetAppDetails.addAppIdName(1, 'Binary.com')).to.equal('');
+            expect(GetAppDetails.addAppIdName(getAppId(), 'Binary.com')).to.equal('');
```

Comment:
why? getAppId won't always return 1, unless the host is binary.com

Reply:
yep, but if you add your app_id then this test fails